### ISSUE TITLE
Change MEV Description text to more ethos-affirming language

### ISSUE THREAD

**Issue by superphiz** [rocket-pool/smartnode#250]
in `rocketpool service config` > `MEV-Boost Settings` > `Description`, there is an item for each relay that reads `Uses Address Blacklist`, where the optimal scenario is that we would see a "NO" in that field, many people have that innate need to "check all the boxes" and will look for something that says "YES". I suggest modifying the text `Uses Address Blacklist` to `Censorship Free` so that ethos-aligned relays will have an affirmative response.

As a second consideration `Includes Frontrunning` has a similar issue: we don't necessarily WANT people to frontrun transactions as this is seen as unethical. It may be worth having a discussion to change this to `Only Extracts Ethical MEV`, so the affirmative response is a healthy response. I'm not sure about the lines between frontrunning and ethical MEV, so I acknowledge that others will want to consider this carefully.

tl;dr: The "best" MEV relays should report `YES` to ethos-aligned criteria.

**Comment by jclapis** [comment]
This whole thing is quite contentious and came out of a long discussion with the community - that being said, I'm going to be revamping it into "categories" based on these parameters instead of just listing all the relays in v1.7, which will incorporate feedback like this. I'll make the categories as clear as possible so people know what they're signing up for.
